challenge-13/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,199 @@
+	// TODO: Start a transaction
+	// TODO: For each product ID in the updates map, update its quantity
+	// TODO: If any update fails, roll back the transaction
+	// TODO: Otherwise, commit the transaction
+	tx, err := ps.db.Begin()
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if err != nil {
+			tx.Rollback()
```

golangci-lint (2.13.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)